pom.xml

```diff
@@ -24,11 +24,14 @@
     <!-- Avoid the message "[WARNING] Using platform encoding (UTF-8 actually) ... also for the failsafe plugin -->
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-    <!-- we use java 1.8 for our projects -->
-    <version.java>1.8</version.java>
+    <!-- we use Java 11 for our projects -->
+    <version.java>11</version.java>
 
     <plugin.version.license>3.0</plugin.version.license>
     <plugin.version.nexus-staging>1.6.13</plugin.version.nexus-staging>
+    <plugin.version.compiler>3.6.2</plugin.version.compiler>
+    <plugin.version.javadoc>3.0.1</plugin.version.javadoc>
```

Comment:
I found that we define the same version in the [platform pom.xml](https://github.com/camunda/camunda-bpm-platform/blob/2204b2830bef534796c42fb8df4f28673c5e406f/pom.xml#L36). We can remove it there. Maybe there are other projects that we can simplify by removing the version there?

Reply:
👍